### PR TITLE
Adds TeamSkeet xPath scraper

### PR DESCRIPTION
**Pull request by halorrr**

### CODE REVIEW

File: scrapers/TeamSkeet.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: //div[contains(@class,'sceneTitle')]/text()
+      Date:
+        selector: //div[contains(@class,'sceneDate')]/text()
+        parseDate: 01/02/2006
+      Details: //div[contains(@class,'sceneDesc')]/text()
+      Performers:
+        Name: //div[contains(@class,'contentTitle')]/a/text()[2]
```
Comment (bnkai) [inline scrapers/TeamSkeet.yml:16]: `/text()[2]` is not needed

File: scrapers/TeamSkeet.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+      Studio:
+        Name: //div[contains(@class,'siteName')]/a/text()[2]
```
Comment (bnkai) [inline scrapers/TeamSkeet.yml:18]: `/text()[2]` is not needed and breaks studios in some samples i found

**Comment by bnkai** [comment]
Also is there a  need for `contains(@class, ...` ?
From what i see the classes are fixed and checking for equality will always be faster than contains.
title `//div[@class="col-12 sceneTitle"]`
date  `//div[@class="col-6 sceneDate"]`
details `//div[@class="col-12 sceneDesc"]`
performer `//div[@class="col-12 contentTitle"]`
studio `//div[@class="siteName float-right px-3 rounded "]`

**Comment by halorrr** [comment]
Updated!

**Comment by bnkai** [comment]
the last changes where only supposed to be for the leftmost part
anyway it should be like this
```
name: "TeamSkeet"
sceneByURL:
  - action: scrapeXPath
    url:
      - https://www.teamskeet.com/movies/
    scraper: sceneScraper
xPathScrapers:
  sceneScraper:
    scene:
      Title: //div[@class="col-12 sceneTitle"]/text()
      Date:
        selector: //div[@class="col-6 sceneDate"]/text()
        parseDate: 01/02/2006
      Details: //div[@class="col-12 sceneDesc"]/text()
      Performers:
        Name: //div[@class="col-12 contentTitle"]/a
      Studio:
        Name: //div[@class="siteName float-right px-3 rounded "]/a
      Image: //stream/@poster

# Last Updated May 17, 2020
```

**Comment by halorrr** [comment]
Updated!

**Pull request merged**